documentation/3_Layers_of_the_Reference_Architecture_Model/3_2_Functional_Layer/3_2_FunctionalLayer.md

```diff
@@ -137,16 +137,19 @@ given in the [Process Layer](../3_4_Process_Layer/3_4_Process_Layer.md).
 
 #### Vocabularies ####
 
-To create and structure metadata, the operator of a Connector may use
-vocabularies. In doing so, an operator of a Connector can use existing
-vocabularies, create own vocabularies, or work with other operators on
-new vocabularies provided by vocabulary hubs. Vocabulary hubs are
-central servers that store vocabularies and enable collaboration.
-Collaboration may comprise search, selection, matching, updating,
-requests for changes, version management, deletion, duplicate
-identification, and unused vocabularies. Vocabulary hubs need to be
-managed. More information about vocabularies is given in the
-[Information Layer](../3_3_Information_Layer/3_3_InformationLayer.md).
+To create and structure metadata, the data provider in a data space may use vocabularies to be able to tell potential consumers of his data assets the semantics of the various data elements in the data assets at a universal location in a standard electronic format.
```

Comment:
Wow that’s a long sentence trying to express a number of ideas, can we make that more readable? 

eg:
To create and structure metadata, the data provider can use vocabularies to define the semantics of the various elements in the data assets. 
The vocabulary in turn can be stored in a Vocabulary Hub that is accessible to all users of a data space in an agreed format.